salsah/src/public/js/00_init_javascript.js

```diff
@@ -313,3 +313,11 @@ var SALSAH_API_LEGACY = {
 
 };
 
+/*
+ * work out an ontology's short name out of its URI
+ * arg id : ontology's URI, for example: `http://www.knora.org/ontology/0103/theatre-societe`
+ * return: the very last part of the URI, in this example: `theatre-societe`
+ */
+SALSAH.vocabularyId2shortName = function(ontologyid) {
+	return ontologyid.substr(ontologyid.lastIndexOf('/') + 1)
```

Comment:
I do not understand why this would be necessary: this does not get the shortcode but the sortname. A the shortname already exists as a member of the answer format.

Reply:
This looks good, as now we can have multiple ontologies per project, e.g., `http://www.knora.org/ontology/0103/theatre-societe` and `http://www.knora.org/ontology/0103/somethingelse`.

Reply:
<img width="749" alt="capture d ecran 2017-12-08 a 13 24 16" src="https://user-images.githubusercontent.com/3933654/33765956-9c812e50-dc1b-11e7-98d9-c05199b6212d.png">

yes it works! would you mind acknowledging @subotic ?